modules/hisilicon/hisi_play.c

```diff
@@ -0,0 +1,157 @@
+/**
+ * @file hisi_play.c  HiSilicon sound driver - player
+ *
+ * Copyright (C) 2022 Dmitry Ilyin
+ */
+
+#include <re.h>
+#include <rem.h>
+#include <baresip.h>
+#include <threads.h>
+
+#include "hisi.h"
+
+#include <mpi_audio.h>
+#include <mpi_sys.h>
+
+struct auplay_st {
+	thrd_t thread;
+	volatile bool run;
+	void *sampv;
```

Comment:
which sample format is used here ?

if 16-bit signed then perhaps change to `int16_t *sampv` ?